client/src/components/HomePage/Slide/index.js

```diff
@@ -45,7 +45,7 @@ class Slide extends React.Component {
         title: 'Animals',
       },
       {
-        link: 'https://imgur.com/n10hQDa.png',
+        link: 'http://sustainability.standardbank.com/image/environmental_blue_icon.png',
         title: 'Environment',
```

Comment:
@OnsNader If it's allowed, you might want to consider saving these images into this repo and referring to those image urls.  Then the VA won't make a broken image icon appear on ChariZone when they move their file to a different path 😄